### PR TITLE
feature: share article through social links

### PR DESCRIPTION
**Pull request by rixcian**
I added functionality to social share links as described in #93 issue.

### CODE REVIEW

File: components/ShareLinks.tsx [hunk 1]
```diff
@@ -1,42 +1,90 @@
 // import Header from '../../components/Header';
-import { Link, Flex, chakra, Icon } from '@chakra-ui/react';
+import copy from 'copy-to-clipboard';
+import { useEffect, useState } from 'react';
+import { Link, Flex, chakra, Icon, Box } from '@chakra-ui/react';
```
Comment (zberwaldt) [inline components/ShareLinks.tsx:4]:
```suggestion
import { Link } from './Link';
```

**Review by zberwaldt — CHANGES_REQUESTED** [review]
In `components/ShareLinks.tsx` to use a wrapped link created by @jakeacooley.

**Comment by rixcian** [comment]
@zberwaldt resolved

**Review by zberwaldt — APPROVED** [review]
Thank you. LGTM.

Thank you for all your work on this.

**Pull request merged**